packages/genesys-spark-components/src/components/stable/gux-listbox-multi/gux-option-multi/gux-option-multi.tsx

```diff
@@ -77,6 +83,10 @@ export class GuxOptionMulti {
     }
   }
 
+  componentWillRender(): void {
+    this.hasSubtext = hasSlot(this.root, 'subtext');
+  }
```

Comment:
If hasSubtext is kept as a state variable, then this probably isn't a good idea. My suggestion to move it to componentWillRender was only if it stopped being a state variable. Otherwise it leads to a possible infinite render loop as changing a state variable calls the re-render process, which then calls this function possibly changing the state again.

Reply:
Thanks for flagging this. Updated now